packages/flame/src/FormField/FormField.tsx

```diff
@@ -29,7 +29,7 @@ const FormHelper: React.FC<FormHelperProps> = ({ status, children, ...restProps
 
 export type BaseLabelProps = React.HTMLAttributes<HTMLLabelElement> & TextProps & { css?: any };
 const BaseLabel = styled(Text)<BaseLabelProps>`
-  display: flex;
+  display: inline-flex;
```

Comment:
Oh, because of here. But why was this done as part of this PR?

Reply:
Labels should be inlined, not block by default.

This is to match up with regular styling.